packages/react-icons/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spnels-fluent-system-test0000",
+  "version": "1.1.50",
+  "description": "Fluent System Icons are a collection of familiar, friendly, and modern icons from Microsoft.",
+  "author": "Microsoft Inc.",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-system-icons.git"
+  },
+  "scripts": {
+    "clean": "shx rm -rf ./components",
+    "cleanSvg": "shx rm -rf ./intermediate",
+    "copy": "node ../../importer/generate.js --source=../../assets --dest=./intermediate --extension=svg --target=react",
+    "convert": "npx @svgr/cli --no-prettier --template template.js --expand-props none --typescript --replace-attr-values \"#212121={props.primary}\" -d ./components ./intermediate/",
+    "optimize": "svgo --folder=./intermediate --precision=2 --disable=removeViewBox,mergePaths",
+    "unfill": "for file in ./components/*.tsx; do sed -i '.bak' 's/ fill=\"none\"//' $file; rm $file.bak; done",
+    "build": "npm run copy && npm run updatePrefix && npm run optimize && npm run convert && npm run unfill && npm run updateExport && npm run cleanSvg",
+    "prepublishOnly": "npm run clean && npm run cleanSvg && npm run build && npm run cleanSvg",
+    "postpublish": "npm run clean",
+    "updatePrefix": "for file in ./intermediate/ic*.svg; do mv \"$file\" \"${file/ic_fluent_/}\"; done",
+    "updateExport": "for file in ./components/*.tsx; do sed -i '.bak' 's/export const/export/' $file; rm $file.bak; done"
+  },
+  "devDependencies": {
+    "@svgr/cli": "^5.5.0",
+    "shx": "^0.3.2",
```

Comment:
Is `shx` as a devDependency needed? It appears that it's only used as a wrapper on the clean commands and may be we can get rid of it. I'm not sure what terminal environments you run this so it could be fine, and if so please ignore this comment.

Reply:
I used shx in the svg icons package to allow it to run on Windows. The official build is on Linux, but it was more convenient during dev time I thought.

Reply:
On that note, @bheston can you confirm which of these commands might not reasonably work for someone trying to maintain this on a Windows machine? I've always started my Windows dev boxes by installing gnu utils so don't recall which tools like sed/rm/whatever aren't readily available.

Reply:
Due to the open source nature of the icons package, it sounds fine to use to allow someone without the gnu utils to clone and use fully the repo. Thanks for the explanation Brian. 

In that case you are using `rm` and `mv` on the `updatePrefix` and `updateExport` scripts without shx, so those commands may not work on Windows :)

Reply:
In digging around and playing with it, I can't find a way to make it work well on both systems. We either hit issues with wildcard expansion or the variations on looping through the files break on one system or the other. Given my lack of time I'm going to suggest we instead tell Windows devs that want to copy/maintain the pipeline use WSL and we will try to just stick to a Ubuntu-friendly solution as that's what the GitHub workflows use to run them anyway.